src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
The WFH calendar is (in the eyes of Symphono employees) only writeable via this command's interface, so I think you were right to include clearing functionality because if I set a far future WFH, I can literally only clear it through this command.  However, it might be a confusing user experience (UX) to provide two different ways to clear a WFH event. Consider this flow:
1. User types `/wfh` -> server says "WFH created for today".
2. User types `/wfh` -> server says "WFH deleted for today."
3. User types `/wfh tomorrow` -> server says "WFH created for tomorrow".
4. User types `/wfh tomorrow` -> server says "WFH created for tomorrow."

See how the experiences are "asymmetrical"? Rather, it might be more helpful to tell the user they're already on the calendar for that day:

1. User types `/wfh` -> server says "WFH created for today"
2. User types `/wfh` -> server says "You're already WFH today"
3. User types `/wfh clear` -> server says "WFH deleted for today"
4. User types `/wfh tomorrow` -> server says "WFH created for tomorrow"
5. User types `/wfh tomorrow` -> server says "You're already WFH tomorrow"
6. User types `/wfh tomorrow clear` -> server says "WFH deleted for tomorrow"